great_expectations/datasource/data_connector/inferred_asset_azure_data_connector.py

```diff
@@ -13,17 +13,10 @@
     sanitize_prefix,
 )
 from great_expectations.execution_engine import ExecutionEngine  # noqa: TCH001
+from great_expectations.optional_imports import BlobServiceClient
```

Comment:
Same comment re check import vs try/except

Reply:
This applies to all of the data_connectors that use the try/except pattern

Reply:
@anthonyburdi Valid comment -- thank you! I solved it differently (using our knowledge of the NotImported internals) -- I think it is OK, given that it is the V3 code. Happy to discuss. Thanks.